silk-react-components/src/HierarchicalMapping/Components/MappingsObject.jsx

```diff
@@ -132,12 +101,17 @@ const MappingsObject = React.createClass({
         if (this.state.expanded) {
             content = (
                 <ObjectRule
+                    // TODO: values are injected twice atm, one can be removed later.
```

Comment:
Can the TODO be resolved?

Reply:
Todo resolved, but for full fix, I need to add abstraction(Wrapper component, which extends base functionality from another component) for ObjectRule. It will cause new issues, so I think for now it's look fine